scripts/create_cluster_for_scale_test.sh

```diff
@@ -0,0 +1,122 @@
+# This script uses eksctl to create an EKS cluster in a new VPC and configures it so that it can be used for scale
+# testing. It is highly recommended to read closely this script and its dependent resources in order to understand
+# what it does.
+# This script does the following:
+# - creates a job execution role. The ARN of this role is printed at the end of the script because this ARN needs to be
+#   pasted into job_config_parameters.py
+# - creates an EKS cluster using the configuration in resources/cluster_in_new_VPC.yaml
+# - installs fluent bit on the EKS cluster, which writes all pod logs to cloudwatch logs under 3 log groups with
+#   prefix /aws/containerinsights/${CLUSTER_NAME}. It can be very useful to have all pod logs available in cloudwatch
```

Comment:
We can also add this documentation link for container insights - https://docs.aws.amazon.com/AmazonCloudWatch/latest/monitoring/ContainerInsights.html

Reply:
added